cpp/src/centrality/betweenness_centrality.cu

```diff
@@ -153,6 +172,79 @@ void BC<VT, ET, WT, result_t>::accumulate(result_t *betweenness,
                     thrust::plus<result_t>());
 }
 
+template <typename VT, typename ET, typename WT, typename result_t>
+__global__ void edges_accumulation_kernel(result_t *betweenness,
+                                          VT number_vertices,
+                                          VT const *indices,
+                                          ET const *offsets,
+                                          VT *distances,
+                                          double *sp_counters,
+                                          double *deltas,
+                                          VT source,
+                                          VT depth)
+{
+  for (int tid = blockIdx.x * blockDim.x + threadIdx.x; tid < number_vertices;
```

Comment:
Right now this is load balanced over a vertex which is not pretty good for performance. Could add a comment here so that forAllEdge type primitive can be used here.

Reply:
Updated in 7ee6a01